src/services/shipment_service.js

```diff
@@ -205,7 +205,11 @@ export default (easypostClient) =>
      */
     static async lowestSmartRate(id, deliveryDays, deliveryAccuracy) {
       const smartRates = await this.getSmartRates(id);
-      return Util.getLowestSmartRate(smartRates, deliveryDays, deliveryAccuracy.toLowerCase());
+      return new EasyPostClient('fake_api_key').Utils.getLowestSmartRate(
```

Comment:
issue: this is still using the EasyPost client

Reply:
Just fixed